scripts/cancel_github_workflows.py

```diff
@@ -23,13 +23,13 @@
   export GITHUB_TOKEN=394ba3b48494ab8f930fbc93
```

Comment:
Just wondering Is this a real github token here :) ?

Reply:
Nope. I just wrote randomly typed one that look real.

Reply:
A real token has 40 characters, this one has 24.